packages/react-tooltip/src/components/Tooltip/useTooltip.tsx

```diff
@@ -23,15 +23,16 @@ import { arrowHeight, tooltipBorderRadius } from './private/constants';
  * @param props - props from this instance of Tooltip
  * @param ref - reference to root HTMLElement of Tooltip
  */
-export const useTooltip_unstable = (props: TooltipProps, ref: React.Ref<HTMLDivElement>): TooltipState => {
+export const useTooltip_unstable = (props: TooltipProps, ref: React.Ref<HTMLElement>): TooltipState => {
```

Comment:
Ditto: there's nothing to forward the ref to (also remove `@param ref` from the comment above)
```suggestion
export const useTooltip_unstable = (props: TooltipProps): TooltipState => {
```

Reply:
Done in 61307b6